CDTS Blazor/Pages/IdentityDocuments.razor

```diff
@@ -32,17 +32,44 @@
         </div>
     </form>
 
+    @if (completed == 1)
+    {
+        <div class="alert alert-success" role="alert">
+            Files uploaded successfully
```

Comment:
Consider using localized resources so we can easily change from English to French.

Reply:
Do we have that setup?

Reply:
Yup we have a lables_en.resx file, where you can store your strings in. We don't have localization yet, but if we get into the habit of doing it now it will be easier down the road.